test/interceptors/dio_interceptor_test.dart

```diff
@@ -44,7 +44,10 @@ void main() {
             500,
             AdapterError(
               type: type,
-              response: Response(statusCode: 500),
+              response: Response(
+                statusCode: 500,
+                request: RequestOptions(path: '/foo'),
```

Comment:
A bit of nitpicking but maybe passing constant declared at the beginning of the file - `path` will make more sense in this case just for the sake of consistency, though it essentially changes nothing.